geowebcache/nestedstorage/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
```

Comment:
If this helps keep things clean I can look at that, no problem.  To me it all goes together but obviously I am trying to learn how this project collaborates so happy to make that change.  Long time developer, new to git/github community projects so might end up asking a few questions on best practice.